src/mpm/canmodel.py

```diff
@@ -595,19 +595,16 @@ def multiplexer_id_nodes(self):
 
     def sort_multiplexer_ids(self) -> None:
         """
-        Sort children by multiplexer ID
+        Sort children by multiplexer ID. This does not update the tree
+        view so the GUI has to be relaunched to display parameters
+        in new order.
 
         Args:
             self: self-instance of this MultiplexedMessage.
         Returns:
             None
         """
-        children_copy = self.children.copy()
-        children_copy.sort(key=lambda x: node_multiplexer_id(x))
-        # Re-add children to trigger the tree update
-        self.recursively_remove_children()
-        for child in children_copy:
-            self.append_child(child)
+        self.children.sort(key=lambda x: node_multiplexer_id(x))
```

Comment:
I had to revert this because the children within multiplex got lost in the old version. This is not great but I think it will do if you just know that the GUI has to be relaunched after sorting+saving

Reply:
I don't want to spend too much effort on this